bin/u-boot-test-console

```diff
@@ -32,4 +32,9 @@ if [ -z "${console_baud}" ]; then
     console_baud=115200
 fi
 
+# Force a reset into FEL mode
+if [ "${recovery_impl}" == "sunxi-fel" ]; then
```

Comment:
I still see this FEL-specific logic in bin/u-boot-test-console, both in the latest version of this pull request, and in branch add-sunxi-fel in your repo.